src/ints/mouse.cpp

```diff
@@ -453,14 +453,14 @@ void MouseQueue::StartTimerIfNeeded()
     bool timer_needed = false;
     uint8_t delay_ms  = UINT8_MAX;
 
-    if (delay.ps2_ms) {
+    if (HasEventPS2()) {
```

Comment:
(out of scope), but maybe:

`constexpr uint8_t max_delay_ms = UINT8_MAX;`

Reply:
The UINT8_MAX is actually a dummy value, it will never be used - added a comment.

Reply:
Ahh; didn't realize that! (I wasn't sure if the per-device delays could grow larger).  Thanks for clarifying.